src/gui/accountstate.h

```diff
@@ -104,6 +106,9 @@ class AccountState : public QObject, public QSharedData
     State state() const;
     static QString stateString(State state);
 
+    int retryCount() const;
```

Comment:
@mgallien Why do you need this public method when only calling it inside this class?

Reply:
made them private
better have those methods to allow better encapsulation